src/modules/facility-loan/dto/get-facility-loan-response.dto.ts

```diff
@@ -0,0 +1,71 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { DateOnlyString } from '@ukef/helpers';
+
+export type GetFacilityLoanResponseDto = GetFacilityLoanResponseItem[];
+
+export class GetFacilityLoanResponseItem {
+  @ApiProperty({
+    description: 'The identifier of the portfolio in ACBS. The UKEF portfolio is E1. The maximum number of characters allowed is 2.',
+  })
+  readonly portfolioIdentifier: string;
+
+  @ApiProperty({ description: 'The identifier of the loan in ACBS.' }) // TODO APIM-126: how many characters will this be? Mulesoft docs say max 10 but ACBS says 0-9. Postman collection has 9 in the example.
```

Comment:
@avaitonis see TODO

Reply:
These IDs are generated by ACBS and are 9 digit long strings. Current IDs in DEV looks like this 000210112

Reply:
Added to description.